src/course-unit/hooks.jsx

```diff
@@ -150,6 +156,17 @@ export const useCourseUnit = ({ courseId, blockId }) => {
     navigate(`/course/${courseId}/container/${movedXBlockParams.targetParentLocator}`);
   };
 
+  const receiveMessage = useCallback(({ data }) => {
+    const { payload, type } = data;
+
+    if (type === messageTypes.handleViewXBlockContent) {
+      const newUnitId = payload.destination.split('/').pop();
+      navigate(`/course/${courseId}/container/${newUnitId}/${sequenceId}`);
+    }
+  }, []);
```

Comment:
Doesn't this callback depend on `sequenceId` and `courseId`?

Reply:
Yes, thanks